wire/msgtransactionnotfound.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2015 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"github.com/kaspanet/kaspad/util/daghash"
+)
+
+// MsgTransactionNotFound defines a kaspa transactionnotfound message which is sent in response to
+// a requesttransactions message if any of the requested data in not available on the peer.
+type MsgTransactionNotFound struct {
+	ID *daghash.TxID
+}
+
+// Command returns the protocol command string for the message. This is part
+// of the Message interface implementation.
+func (msg *MsgTransactionNotFound) Command() MessageCommand {
+	return CmdNotFound
+}
+
+// NewMsgTransactionNotFound returns a new kaspa transactionsnotfound message that conforms to the
```

Comment:
Change transactionsnotfound->TransactionNotFound

Reply:
I wrote transactionsnotfound because that's what's going on in rest of wire package.

Anyway, fixed here, because it looks better, even if non-compliant.